src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestIT.java

```diff
@@ -64,7 +64,7 @@ public void beforeAll() throws Exception {
         .execute(String.format("create or replace schema %s;", TEST_SCHEMA));
     jdbcConnection
         .createStatement()
-        .execute(String.format("create or replace table %s (c1 char(10));", TEST_TABLE));
+        .execute(String.format("create or replace table %s (c1 variant);", TEST_TABLE));
```

Comment:
Could you give a quick explanation on some of the changes in this file? Like why the data type change from char to variant, maybe we should add new tests if you're trying to test different scenarios?

Reply:
thanks, leftover, reverted